src/modules/RA/DataLayer/components/tabs/StyleTab/StyleEditor.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+
+import { useTranslate, RadioButtonGroupInput } from 'react-admin';
+import { makeStyles } from '@material-ui/core/styles';
+
+import { getShapeFromGeomType } from '../../../../../../utils/geom';
+import Condition from '../../../../../../components/react-admin/Condition';
+
+import WizardFill from './Style/WizardFill';
+import WizardFillExtrusion from './Style/WizardFillExtrusion';
+import WizardLine from './Style/WizardLine';
+import WizardCircle from './Style/WizardCircle';
+
+import ExpertStyleField from './ExpertStyleField';
+
+import styles from './Style/styles';
+
+const useStyles = makeStyles(styles);
+
+const StyleEditor = ({ path, geomType, fields, getValuesOfProperty }) => {
+  const classes = useStyles();
+  const translate = useTranslate();
+
+  const styleType = getShapeFromGeomType(geomType);
+
+  return (
+    <div className={classes.styleEditor}>
+      <div
+        className="wizard-select"
+      >
+        <RadioButtonGroupInput
+          source={`${path}.type`}
+          label={translate('style-editor.type.input')}
+          choices={[
+            { id: 'wizard', name: translate('style-editor.type.wizard') },
+            { id: 'advanced', name: translate('style-editor.type.expert') },
+          ]}
+          initialValue="wizard"
+          helperText={false}
+        />
+      </div>
+
+      {styleType === 'fill' && <h1>{translate('style-editor.geometry')} {translate('style-editor.polygon')}</h1>}
+      {styleType === 'line' && <h1>{translate('style-editor.geometry')} {translate('style-editor.line-type')}</h1>}
+      {styleType === 'circle' && <h1>{translate('style-editor.geometry')} {translate('style-editor.point')}</h1>}
+
+      {!['fill', 'line', 'circle'].includes(styleType) && (
+        <h2>{translate('style-editor.other', { type: styleType })}</h2>
+      )}
```

Comment:
Is this relevant to change the tag when the type is unknown?
If not, maybe you can avoid some LOC
```jsx
<h1>
  {['fill', 'line', 'circle'].includes(styleType) 
    ? <>{translate('style-editor.geometry')}</>
    : <>{translate('style-editor.other', { type: styleType })}</>
  }
  {styleType === 'fill' && <>{translate('style-editor.polygon')}</>}
  {styleType === 'line' && <>{translate('style-editor.line')}</>}
  {styleType === 'circle' && <>{translate('style-editor.point')}</>}
</h1>
```

Reply:
Indeed. Done !